app/2.0/toolbox/lazy-imports.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Lazy Imports
+---
+
+<!-- toc -->
+
+Normal HTML Imports are eager, meaning that they are loaded and evaluated in
+order first, before any code that follows. You can get a large performance
+improvement by lazily loading code at runtime, so that you only load the
+minimal amount of code needed to display the current view. This is a key piece
+of [the PRPL pattern](/2.0/toolbox/prpl).
+
+
+Lazy imports are an optional method for adding declarative lazy-loaded imports. The Polymer CLI
+can use lazy imports to provide better analysis—for example, providing appropriate lint errors for
+lazy-loaded elements.
+
+## Using lazy imports
+
+The lazy imports mechanism is already built in to the latest version of Polymer starter kit
```

Comment:
"latest version of Polymer starter kit" --> "latest version of the Polymer starter kit"